### PR TITLE
Remove mctaylors.ru mentions

### PR DESCRIPTION
**Pull request by mctaylors**
Dear @Octol1ttle,

Please take this PR completely seriously.

Due to your recent decisions, you have decided to leave mctaylors.ru. Unfortunately for you, Octobot's lifespan can no longer continue. SSH keys of github-actions will be removed.

You also mentioned the archiving of this repository. As such, we and a few contributors, in view of the LICENSE, have every right to fork this repository (if necessary) with the right to MODIFICATION and PRIVATE USE.

Thank you for your cooperation.

### CODE REVIEW

**Review by neroduckale — APPROVED** [review]

File: .github/workflows/build-push.yml
Comment (Octol1ttle) [inline .github/workflows/build-push.yml:1]: As this CI/CD implementation isn't hardcoded to use `mctaylors.ru` (the workflow receives this information via environments, which have to be deleted in the settings), it should be kept, at least as a reference for other people looking for a simple SSH-based CI/CD

File: docs/README.md
Comment (Octol1ttle) [inline docs/README.md:1]: Reference to `mctaylors.ru` on line 2

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

File: .github/workflows/build-push.yml [hunk 1]
```diff
@@ -61,4 +61,4 @@ jobs:
         env:
           SSH_USER: ${{secrets.SSH_USER}}
           SSH_HOST: ${{secrets.SSH_HOST}}
-          START_COMMAND: ${{vars.START_COMMAND}}
+          START_COMMAND: ${{vars.START_COMMAND}}
```
Comment (Octol1ttle) [inline .github/workflows/build-push.yml:64]: revert

File: docs/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 </p>
 
-<a href="https://github.com/LabsDevelopment/Octobot/blob/master/LICENSE"><img src="https://img.shields.io/github/license/LabsDevelopment/Octobot?logo=git"></img></a>
+<a href="https://github.com/TeamOctolings/Octobot/blob/master/LICENSE"><img src="https://img.shields.io/github/license/TeamOctolings/Octobot?logo=git"></img></a>
 <a href="https://github.com/Remora/Remora.Discord"><img src="https://img.shields.io/badge/powered_by-Remora.Discord-blue"></img></a>
-<a href="https://github.com/LabsDevelopment/Octobot/commit/master"><img src="https://img.shields.io/github/last-commit/LabsDevelopment/Octobot?logo=github"></img></a>
+<a href="https://github.com/TeamOctolings/Octobot/commit/master"><img src="https://img.shields.io/github/last-commit/TeamOctolings/Octobot?logo=github"></img></a>
 
-Veemo! I'm a general-purpose bot for moderation (formerly known as Boyfriend) written by [Labs Development Team](https://github.com/LabsDevelopment) in C# and Remora.Discord
+Veemo! I'm a general-purpose bot for moderation (formerly known as Boyfriend) written by [Labs Development Team](https://github.com/TeamOctolings) in C# and Remora.Discord
```
Comment (Octol1ttle) [inline docs/README.md:9]: > Labs Development Team

File: .github/workflows/build-push.yml
Comment (Octol1ttle) [inline .github/workflows/build-push.yml:1]: The "revert" call was about the removal of the newline at the end of file 😁

Comment (mctaylors) [reply .github/workflows/build-push.yml:1]: Then please always clarify what you mean by "revert". One simple word can have several meanings.

**Pull request merged**